test_runner/src/main/kotlin/ftl/reports/util/ReportManager.kt

```diff
@@ -96,13 +98,53 @@ object ReportManager {
         return matrices.exitCode()
     }
 
+    data class ShardEfficiency(
+        val expectedTime: Double,
+        val finalTime: Double,
+        val efficiency: Double
+    )
+
+    fun createShardEfficiencyList(oldResult: JUnitTestResult, newResult: JUnitTestResult, args: IArgs):
+            List<ShardEfficiency> {
+        val oldJunitMap = Shard.createJunitMap(oldResult, args)
+        val newJunitMap = Shard.createJunitMap(newResult, args)
+
+        val timeList = mutableListOf<ShardEfficiency>()
+        args.testShardChunks.forEach { testSuite ->
+
+            var expectedTime = 0.0
+            var finalTime = 0.0
+            testSuite.forEach { testCase ->
+                expectedTime += oldJunitMap[testCase] ?: 0.0
+                finalTime += newJunitMap[testCase] ?: 0.0
+            }
+
+            val efficiency = 100 - (expectedTime * 100.0 / finalTime)
```

Comment:
if you do it this way, it would be (finalTime * 100 / expectedTime) - 100
Ex. expected 10 sec, actual 9 sec
(9 * 100.0 / 10) - 100 = -10%   -----> You were 10% off and it was to the downside 
(11 * 100.0 / 10) - 100 = +10% -----> You were 10% off and it was to the upside 

Or a more readable error calc imo would be:
(finalTime - expectedTime) / expectedTime * 100
(9-10)/10*100 = -10%
(11-10)/10*100 = +10%